contracts/src/fp/AccessManager.sol

```diff
@@ -26,6 +26,28 @@ contract AccessManager is Ownable {
     /// @notice Tracks whitelisted challengers.
     mapping(address => bool) public challengers;
 
+    /// @notice The timeout (in seconds) after which permissionless proposing is allowed (immutable).
+    uint256 public immutable permissionlessTimeout;
```

Comment:
Why is this immutable while `fallbackProposalTimeout` in the L2OO is not immutable?